test/logflare/source_schemas_test.exs

```diff
@@ -4,6 +4,36 @@ defmodule Logflare.SourceSchemasTest do
 
   alias Logflare.SourceSchemas
 
+  test "create_source_schema/2 with valid attributes" do
+    user = insert(:user)
+    source = insert(:source, user: user)
+
+    attrs = %{
+      bigquery_schema: TestUtils.build_bq_schema(%{"test" => 123}),
+      schema_flat_map: %{"test" => "integer"}
```

Comment:
can we also add a test to cover validation for nested maps, it should reject multiple levels

Reply:
I will do it today :pray: